crates/ruff_server/src/server/api/requests/code_action.rs

```diff
@@ -140,7 +140,7 @@ fn organize_imports(snapshot: &DocumentSnapshot) -> crate::Result<CodeActionOrCo
                 document,
                 snapshot.resolved_client_capabilities(),
                 snapshot.url(),
-                &snapshot.settings().linter,
+                snapshot.settings().linter(),
```

Comment:
These getters are perhaps a nice example of something that could be put up as a trivial standalone PR that would be immediately approved and merged :)

Reply:
Thanks, I'll keep that in mind for the future 👍